### ISSUE TITLE
Harcoded  stuff in preprocess_fds

### ISSUE THREAD

**Issue by chraibi** [JuPedSim/jpsfire#12]
In GitLab by @vanweyenberge1 on Dec 29, 2017, 16:11

I noticed there are still some hard-coded paths and other code in the preprocess_fds.py.

For example, I cannot run the CS1 case in order to obtain the correct files (2_extinction_grids and 3_sfgrids).

**Comment by chraibi** [comment]
In GitLab by @vanweyenberge1 on Dec 29, 2017, 16:12

changed the description

**Comment by chraibi** [comment]
In GitLab by @chraibi on Dec 29, 2017, 16:17

changed the description

**Comment by chraibi** [comment]
In GitLab by @chraibi on Dec 29, 2017, 16:17

I have the same problem.

**Comment by chraibi** [comment]
In GitLab by @chraibi on Dec 29, 2017, 17:25

What error are you getting?

**Comment by chraibi** [comment]
In GitLab by @chraibi on Dec 29, 2017, 17:41

`3_sfgrids` is empty

Call: 

```bash
python preprocess_fds.py -j ../demos/CS1/JuPedSim -f ../demos/CS1/FDS/
>> Running: preprocess_fds.py
>> Done.
>> Logfile: ../demos/CS1/FDS/log.txt
```

![Screen_Shot_2017-12-29_at_17.39.57](/uploads/3d1ddc573ea7b8816b20b4e6631dc56e/Screen_Shot_2017-12-29_at_17.39.57.png)

Looking in the log I see that no exits are treated. Maybe this is the reason why `3_sfgrids` is empty?

[log.txt](/uploads/bc697143cc3143dedfabdae7aef5ace3/log.txt)

**Comment by chraibi** [comment]
In GitLab by @chraibi on Dec 29, 2017, 18:00

It is empty for other demos as well. So must have happened during restructuring.

**Comment by chraibi** [comment]
In GitLab by @chraibi on Dec 29, 2017, 18:21

@vanweyenberge1 which branch are you using?

**Comment by chraibi** [comment]
In GitLab by @chraibi on Dec 29, 2017, 19:09

closed via commit 612c167ee7bac6633fffb1a9be27712bb82e5793

**Comment by chraibi** [comment]
In GitLab by @vanweyenberge1 on Dec 30, 2017, 09:49

I'm on the master branch
